functions.php

```diff
@@ -1033,18 +1033,20 @@ function my_post_admin_notices()
 });
 
 /* Disable Gutenberg for normal posts */
-function mongabay_disable_gutenberg_posts($can_edit, $post_type)
+function mongabay_disable_gutenberg($can_edit, $post)
 {
     $excluded_post_types = array('custom-story');
     $excluded_pages = array('about', 'contact', 'terms');
 
-    return in_array($post_type, $excluded_post_types);
-}
+    if ($post->post_type === 'custom-story') {
+        return true;
+    }
 
-function mongabay_disable_gutenberg_pages($can_edit, $post)
-{
-    $excluded_pages = array('about', 'contact', 'terms');
-    return in_array($post->post_name, $excluded_pages);
+    if (in_array($post->post_name, $excluded_pages)) {

```

Comment:
What if one saves a post with title 'about' or 'contact'? We want to target pages only and separately custom post types.